obsei/source/pandas_source.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from obsei import configuration
+from typing import List, Optional, Dict, Any
+
+from pandas.core.frame import DataFrame
+from obsei.source.base_source import BaseSource, BaseSourceConfig
+from obsei.payload import TextPayload
+from obsei.misc.utils import (
+    DATETIME_STRING_PATTERN,
+    DEFAULT_LOOKUP_PERIOD,
+    convert_utc_time,
+)
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+class PandasSourceConfig(BaseSourceConfig):
+    TYPE: str = "PandasScrapper"
+    target: str = None
+    target_df: Optional[pd.DataFrame] = None
```

Comment:
Let's keep only three variables PandasSinkConfig -
```
class PandasSourceConfig(BaseSinkConfig):
    TYPE: str = "Pandas"
    dataframe: Optional[DataFrame] = None
    # By default it will add all fields to meta
    include_columns_list: Optional[List[str]] = None
    # Text columns
    text_columns_list: List[str]
```

And constructor need to perform validation about text_columns_list

Reply:
I'm refactoring the code for only dataframe. I'll also include required validations.